{"uid":"7a4cb34bd7d9a5f7","name":"CliKintoneTest-78 Should return the error message when the user has no privilege to view records.","fullName":"features/record/export.feature#CliKintoneTest-78 Should return the error message when the user has no privilege to view records.","historyId":"f473ce6099a3904e4fcfa47f7a9a85dd:d41d8cd98f00b204e9800998ecf8427e","time":{"start":1734425738700,"stop":1734425740630,"duration":1930},"description":"","status":"passed","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[],"testStage":{"description":"","status":"passed","steps":[{"name":"Given Load app ID of the app \"app_for_export\" as env var: \"APP_ID\"","time":{"start":1734425738702,"stop":1734425738703,"duration":1},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"And Load username and password of the app \"app_for_export\" with exact permissions \"add\" as env vars: \"USERNAME\" and \"PASSWORD\"","time":{"start":1734425738703,"stop":1734425738703,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"When I run the command with args \"record export --base-url $$TEST_KINTONE_BASE_URL --app $APP_ID --username $USERNAME --password $PASSWORD\"","time":{"start":1734425738703,"stop":1734425740628,"duration":1925},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"Then I should get the exit code is non-zero","time":{"start":1734425740628,"stop":1734425740628,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"And The output error message should match with the pattern: \"ERROR: \\[403] \\[CB_NO02] No privilege to proceed.\"","time":{"start":1734425740628,"stop":1734425740629,"duration":1},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":5,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},"afterStages":[],"labels":[{"name":"host","value":"fv-az1487-799"},{"name":"language","value":"javascript"},{"name":"framework","value":"cucumberjs"},{"name":"thread","value":"2140"},{"name":"feature","value":"cli-kintone export command"},{"name":"suite","value":"CliKintoneTest-78 Should return the error message when the user has no privilege to view records."},{"name":"tag","value":"@isolated"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[],"history":{"statistic":{"failed":0,"broken":0,"skipped":0,"passed":2,"unknown":1,"total":3},"items":[{"uid":"f2f7618a3e8ac79","reportUrl":"https://kintone.github.io/cli-kintone/6757//#testresult/f2f7618a3e8ac79","status":"passed","time":{"start":1734421056499,"stop":1734421058576,"duration":2077}},{"uid":"90831ce50697f62b","reportUrl":"https://kintone.github.io/cli-kintone/6670//#testresult/90831ce50697f62b","status":"unknown","statusDetails":"The test doesn't have an implementation.","time":{"start":1733124270414,"stop":1733124270420,"duration":6}}]},"tags":["@isolated"]},"source":"7a4cb34bd7d9a5f7.json","parameterValues":[]}